docs/guides/key-manager/get-controllers.md

```diff
@@ -112,7 +112,7 @@ console.log(
 // decoded permission for 0x5F606b5b237623463a90F63230F9b929321dbCBa = {
 //   "CHANGEOWNER": true,
 //   "CHANGEPERMISSIONS": true,
-//   "ADDPERMISSIONS": true,
+//   "ADDCONTROLLER": true,
```

Comment:
Should we also add the other permissions ?

Reply:
Yes good point @YamenMerhi 
We should update the comment to include all the permissions except DELEGATECALL + SUPER_DELEGATECALL + REENTRANCY.

```js
// "CHANGEOWNER": true,
// "ADDCONTROLLER": true,
// "CHANGEPERMISSIONS": true,
// "ADDEXTENSIONS": true,
// "CHANGEEXTENSIONS": true,
// "ADDUNIVERSALRECEIVERDELEGATE": true,
// "CHANGEUNIVERSALRECEIVERDELEGATE": true,
// "REENTRANCY": false,
// "SUPER_TRANSFERVALUE": true,
// "TRANSFERVALUE": true,
// "SUPER_CALL": true,
// "CALL": true,
// "SUPER_STATICCALL": true,
// "STATICCALL": true,
// "SUPER_DELEGATECALL": false,
// "DELEGATECALL": false,
// "DEPLOY": true,
// "SUPER_SETDATA": true,
// "SETDATA": true,
// "ENCRYPT": true,
// "DECRYPT": true,
// "SIGN": true,
```